src/PointwiseFunctions/Hydro/EquationsOfState/Python/Enthalpy.cpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Hydro/EquationsOfState/Python/Enthalpy.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.hpp"
```

Comment:
move this include down (keep only the Python/Enthalpy.hpp as the first line here, since it's the header corresponding to this file)